postgres/metadata.csv

```diff
@@ -3,6 +3,7 @@ postgresql.active_queries,gauge,,,,"Enabled with `collect_activity_metrics`. The
 postgresql.active_waiting_queries,gauge,,,,"Enabled with `collect_activity_metrics`. The number of waiting queries in this database in state active. This metric (by default) is tagged with db, app, user.",0,postgres,transactions active_waiting queries,,
 postgresql.activity.backend_xid_age,gauge,,transaction,,"The age of the oldest backend's xid relative to latest stable xid. This metric (by default) is tagged with db, app, user.",-1,postgres,activity xid,,
 postgresql.activity.backend_xmin_age,gauge,,transaction,,"The age of the oldest backend's xmin horizon relative to latest stable xid. This metric (by default) is tagged with db, app, user.",-1,postgres,activity xmin,,
+postgresql.activity.wait_event,gauge,,,,"The number of wait events. This metric is tagged by user. This metric is tagged by user, db, app and backend_type",0,postgres,wait_event,,
```

Comment:
> The number of wait events.

This description sounds misleading to me. The metric shows the number of sessions in a wait event at the time of the snapshot, not the total number of wait events over a time period.